modules/virt-must-gather-options.adoc

```diff
@@ -41,25 +43,29 @@ Each script is compatible only with certain environment variable combinations.
 
 `/usr/bin/gather --images`:: Collect image, image-stream, and image-stream-tags custom resource information. This script is compatible only with the `PROS` variable.
 
+`/usr/bin/gather --instancetypes`:: Collect InstanceTypes information. This information is not currently collected by default; you can, however, optionally collect it.
```

Comment:
Should the `InstanceTypes` field be in backticks?

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference